src/Utilities/UnitTests/ToolLocationHelper_Tests.cs

```diff
@@ -951,8 +974,8 @@ public void GetPathToWindowsSdk()
                                     }
   ";
 
-        [Fact(Skip = "Ignored in MSTest")]
-        // Ignore: Test requires installed toolset.
+#if FEATURE_CODETASKFACTORY
```

Comment:
This was just moved from the next test case, we didn't notice the need for it before because it was skipped.